src/harness/projectsRunner.ts

```diff
@@ -110,6 +110,7 @@ class ProjectRunner extends RunnerBase {
                 else if (url.indexOf(diskProjectPath) === 0) {
                     // Replace the disk specific path into the project root path
                     url = url.substr(diskProjectPath.length);
+                    // TODO: should be '!=='?

```

Comment:
should be safe to remove it.. i would run the tests as well. and please remove the comment